src/System.Memory/Directory.Build.props

```diff
@@ -2,10 +2,10 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <!-- System.Memory has forwarded types into the runtime on netcoreapp/uap
-         It must win over assemblies versioned at 4.0.* -->
-    <AssemblyVersion>4.2.0.0</AssemblyVersion>
+It must win over assemblies versioned at 4.0.* -->
```

Comment:
nit: whitespace.  Also we can just remove this entire comment, it was only relevant when we were shipping the package.